Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1822,24 +1822,48 @@
 #endif
 ))
 
+(define (allow-thread-adopt-exception-handler)
+    (when (defined? 'thread_adopt_exception_handler)
+        (allow mach-message-send (kernel-mig-routine thread_adopt_exception_handler))
+    )
+)
```

Comment:
@pvollan just double checking this works properly on old and new SDK versions given it doesn't have the `#if HAVE(HARDENED_MACH_EXCEPTIONS)`

Reply:
I think it should be ok as that won't be defined.